recipes/paradoc/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "paradoc" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Krande/paradoc/archive/v{{ version }}.tar.gz
+  sha256: b82ee4c9bc990e339f3341559dd9bd464efb3cf3c7105af5d8ef35920a0b03d4
+
+build:
+  skip: true  # [py<310]
```

Comment:
This recipe can probably be OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Reply:
Hey @ocefpaf! Making it noarch was my original plan, but unfortunately I need the conditional windows run requirement `- pywin32 # [win]`. Do you know of a way I can make this noarch and still keep this conditional run requirement for windows users ?

(I don't want windows users having to separately install pywin32 for each paradoc installation )

Reply:
Hey again, @ocefpaf. Looking at the conda-forge FAQ again, it seems I might have missed the `- __win   # [win]` requirement. I have tried building it as a noarch package, but it seems I will also have to modify the conda-forge yaml with 

```
noarch_platforms:
  - linux_64
  - win_64
```

which it seems I am not allowed to do according to the build bot?